### DISCUSSION TITLE
What's the best way to add a new grammar repository to the docs page?

### DISCUSSION THREAD

**Discussion by doyled-it** [tree-sitter/tree-sitter#3198] [Grammars]
My team has recently created [this grammar repository](https://github.com/janus-llm/tree-sitter-ibmhlasm) for IBM HLASM and we'd like to add it to the [main docs page](https://tree-sitter.github.io/tree-sitter/) if possible. Should we just open a PR with that addition to the `index.md` file?

**Comment by sogaiu** [comment]
Looks like others have had success with that approach :)

* https://github.com/tree-sitter/tree-sitter/pull/3114
* https://github.com/tree-sitter/tree-sitter/pull/3189

**Reply by doyled-it** [reply]
Apologies! I thought I searched for it well enough, but I didn't see those when I did. Either way, thanks! I'll do that soon :)
